dali/operators/reader/loader/video/video_test_base.cc

```diff
@@ -66,8 +67,20 @@ void VideoTestBase::LoadFrames(
 }
 
 void VideoTestBase::CompareFrames(const uint8_t *frame, const uint8_t *gt, int size, int eps) {
-    for (int j = 0; j < size; ++j) {
-      ASSERT_NEAR(frame[j], gt[j], eps);
-    }
+  for (int j = 0; j < size; ++j) {
+    ASSERT_NEAR(frame[j], gt[j], eps);
+  }
 }
+
+void VideoTestBase::CompareFramesAvg(
+  const uint8_t *frame, const uint8_t *gt, int size, double eps) {
+  double sum = 0.0;
+  for (int j = 0; j < size; ++j) {
+    sum += std::abs(frame[j]-gt[j]);
```

Comment:
No, for `uint8_t` before subtraction both operands are promoted to `int`, then subtraction is done, then `abs`. I may change that, if you feel that it would improve the readability.

Reply:
A bit of reference: https://www.eskimo.com/~scs/cclass/int/sx4cb.html

Reply:
No. I'm fine with that.